### PR TITLE
Misc enhancements

### PR DESCRIPTION
**Pull request by joneslee85**
cc @notahat

### CODE REVIEW

File: safe_shell.gemspec [hunk 1]
```diff
@@ -11,14 +10,15 @@ Gem::Specification.new do |s|
   s.homepage    = "http://github.com/envato/safe_shell"
   s.summary     = %q{Safely execute shell commands and get their output.}
   s.description = %q{Execute shell commands and get the resulting output, but without the security problems of Ruby’s backtick operator.}
+  s.license     = "MIT"
```
Comment (pablolee) [inline safe_shell.gemspec]: Is there a standard license we use at Envato?

File: safe_shell.gemspec [hunk 2]
```diff
@@ -11,14 +10,15 @@ Gem::Specification.new do |s|
...
 
-  s.rubyforge_project = "safe_shell"
-
   s.files            = `git ls-files`.split("\n")
   s.test_files       = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables      = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths    = ["lib"]
   s.extra_rdoc_files = ["LICENSE", "README.rdoc"]
+
+  s.required_ruby_version = ">= 2.0.0"
```
Comment (pablolee) [inline safe_shell.gemspec]: If we're going to change the required ruby version, is it worth bumping the version of the gem as well?

Comment (joneslee85) [inline safe_shell.gemspec]: good point, we should :+1:

File: safe_shell.gemspec (hunk 1 above)
Comment (joneslee85) [inline safe_shell.gemspec]: the clause in `LICENSE` file match 100% MIT license, so I think it is MIT

Comment (pablolee) [inline safe_shell.gemspec]: Awesome

**Comment by pablolee** [comment]
+1

**Pull request merged**